zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -669,18 +670,15 @@ object Http {
   /**
    * Attempts to retrieve files from the classpath.
    */
-  def getResource(path: String): Http[Any, Throwable, Any, net.URL] =
+  def getResource(path: String): Http[Blocking, Throwable, Any, net.URL] =
```

Comment:
```suggestion
  def getResource(path: String): Http[Any, Throwable, Any, net.URL] =
```
I don't think we need this.